extension/readme/src/views/Report/report.js

```diff
@@ -147,7 +147,7 @@ class Report extends Component {
                       className="text-red-500 text-xs italic"
                     />
                   </div>
-                  <div className="fak">
+                  {/* <div className="fak">
```

Comment:
Why is it commented? Is this something that will be added later or should it be deleted?